sandbox/cgroup/tracking_test.go

```diff
@@ -56,18 +56,20 @@ func (s *trackingSuite) TearDownTest(c *C) {
 	dirs.SetRootDir("")
 }
 
-// CreateTransientScopeForTracking is a no-op when refresh app awareness is off
+// CreateTransientScopeForTracking is not a no-op when refresh app awareness is off
```

Comment:
this comment is now confusing, would recommend:

```suggestion
// CreateTransientScopeForTracking always attempts to track, even when refresh app awareness flag is off
```

Reply:
Applied, thank you!